x/auth/ante/ante_test.go

```diff
@@ -166,25 +166,6 @@ func TestAnteHandlerSigErrors(t *testing.T) {
 			false,
 			sdkerrors.ErrUnknownAddress,
 		},
-		{
-			"save the first account, but second is still unrecognized",
```

Comment:
this test is not valid anymore after the change in behaviours, since the pubkey checking step is done separately during sig verification